### DISCUSSION TITLE
Is `AddWebResourceRequestedFilterWithRequestSourceKinds` available outside of Win32?

### DISCUSSION THREAD

**Discussion by Eilon** [MicrosoftEdge/WebView2Feedback#4888] [General]
In https://github.com/MicrosoftEdge/WebView2Announcements/issues/102 it was posted that `AddWebResourceRequestedFilter` is deprecated and that `AddWebResourceRequestedFilterWithRequestSourceKinds` must be used instead.

The issue links to docs, but only the Win32 docs mention the new API. The other links point to the apparently deprecated API. And when I tried to use the latest pre-release NuGet package (1.0.2895-prerelease), I don't see the API available in WPF or WinUI:

![image](https://github.com/user-attachments/assets/2942f646-9eae-440e-a18a-b321841c55be)

cc @victorhuangwq

**Comment by mungojam** [comment]
Yeah this doc doesn't seem to mention the new one

https://learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/webresourcerequested?tabs=dotnet

**Reply by victorhuangwq** [reply]
@Eilon and @mungojam for .NET and WinRT, you would still be using the `AddWebResourceRequestedFilter` API, but you would have to pass in the third parameter `RequestSourceKinds`

![image](https://github.com/user-attachments/assets/9af6d163-4ca2-43aa-a816-4db42fb797de)

https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter?view=webview2-dotnet-1.0.2849.39

Thanks for the question - I can add a clarification to the announcement.
